src/app/components/create-article/create-article.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from '@angular/core';
+import { FormControl, FormBuilder, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+import { MatChipInputEvent } from '@angular/material/chips';
+
+import { ArticlesService } from "../../services/articles.service";
+
+@Component({
+  selector: 'app-create-article',
+  templateUrl: './create-article.component.html',
+  styleUrls: ['./create-article.component.scss']
+})
+export class CreateArticleComponent {
```

Comment:
let's add here also a map of errors, like error map for elements <ErrorKey, ErrorMessage>